<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Динамические системы</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="QSlider" name="horizontalSlider">
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Первый параметр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Второй параметр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Аттрактор Рёсслера</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Лоренца</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Chen-Lee</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Aizawa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Henon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор HR3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Chua</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Gin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Anishchenko-Astakhov</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Arnedo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Bouali</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Burke-Shaw</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Chen-Celikovsky</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Coullet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Dadras</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Dequan-Li</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Finance</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Four-wing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Genesio-Tesi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Hadley</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Halvorsen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Liu-Chen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Lorenz mod 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Lorenz mod 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Lu-Chen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Newton-Leipnik</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Nose-Hoover</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Qi-Chen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Rayleigh-Benard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Rucklidge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Sakaraya</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Shimizu-Morioka</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Thomas</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор TSUCS1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор TSUCS2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Wang-Sun</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Wimol-Banlue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Аттрактор Yu-Wang</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QComboBox" name="constantsBox"/>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Построить модель</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pauseButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Пауза</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="PointsViewQGLWidget" name="pointsViewQGLWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>370</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Третий параметр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>14.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Настройки</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PointsViewQGLWidget</class>
   <extends>QWidget</extends>
   <header>PointsViewQGLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Window</receiver>
   <slot>slot_restart_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Window</receiver>
   <slot>slot_time_slider(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Window</receiver>
   <slot>slot_model_selection(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>constantsBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Window</receiver>
   <slot>slot_constants_selection(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>Window</receiver>
   <slot>slot_pause_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Window</receiver>
   <slot>slot_open_preferences()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_restart_button()</slot>
  <slot>slot_time_slider(int)</slot>
  <slot>slot_model_selection(QString)</slot>
  <slot>slot_constants_selection(QString)</slot>
  <slot>slot_pause_button()</slot>
  <slot>slot_open_preferences()</slot>
 </slots>
</ui>
